.github/workflows/workflow.yaml

```diff
@@ -2,26 +2,26 @@ name: build_test_release
 on:
   push:
     paths-ignore:
-      - ".github/workflows/website.yaml"
```

Comment:
Please do not change the indentation/formatting, as it is hard to tell what is new code vs. formatting changes.